src/commands/package.ts

```
@@ -54,4 +97,75 @@ export default class Package extends Command {
     )
     return newOrganization
   }
+
+  private async buildMicroServicesDockerImages(
```

I would check for Dockerfile before calling the docker command and fail silently with it. In this way we can give a more explicit error like "Dockerfile not found for microservice <abc>. Please provide one in order to proceed with the bundle packaging"